src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -11,6 +11,9 @@
     "PWA": {
       "longName": "pwa"
     },
+    "AOT": {
+      "longName": "aot"
+    },
```

Comment:
What’s the use case for including this in the template? If it’s mainly to simplify our tests that makes sense. Otherwise I question whether customers would really be thinking about this at project creation time, given that it’s trivial to add later.

Reply:
Also this doesn’t appear to be used in the template csproj at all.

Reply:
Oh shoot, I updated the generated output. But fair enough, I can knock it out.